src/main/java/Duke.java

```diff
@@ -1,10 +1,57 @@
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
 public class Duke {
+    private static final Tasks taskManager = new Tasks();
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greetUser();
+        echo();
+        exit();
+    }
+    private static void greetUser() {
+        System.out.println("________________________________________");
+        System.out.println(" Hello! I'm Judy");
+        System.out.println(" What can I do for you?");
+        System.out.println("________________________________________");
+    }
+
+    private static void echo() {
```

Comment:
Maybe rename this method to better reflect the functionality since it is no longer echoing the user input